extensions/amp-story/1.0/amp-story.js

```diff
@@ -999,6 +1027,9 @@ export class AmpStory extends AMP.BaseElement {
           setAttributeInMutate(oldPage, Attributes.VISITED);
         }
 
+        // Start progress bar update for target page.
+        this.advancement_.updateProgress(targetPageId);
```

Comment:
While I understand why you want to keep updating the progress this way, as it keeps the page-advancement code working as before, the logic becomes a bit weird:
1. `amp-story` triggers `advancement.updateProgress`
2. `advancement.updateProgress` triggers `amp-story` callback

What do you think of calling `story.emitProgress_` directly from here, to avoid that ping pong? There's all the needed data (`targetPage`) so that'd work easily